receiver_custom.html

```diff
@@ -403,7 +406,24 @@ <h3 id= "error_description" class ="cast_text"></h3>
       // prepare the player load script
       var playerJs = document.createElement("script");
       playerJs.type = "text/javascript";
-      playerJs.src = OOYALA_PLAYER_URL + "?" + (debug? ("debug=true&" + v3Version) : v3Version);
+      playerJs.src = OOYALA_PLAYER_URL;
+      if (debug || v3Version) {
+        if (!playerJs.src.match(/\?$/)) {
```

Comment:
This code will append a ? if the url doesn't end with a question mark:
```
"http://url?".match(/\?$/)
["?"]
"http://url?1".match(/\?$/)
null
```
Please remove the $ from your regex
```
"http://url?1".match(/\?/)
["?"]
```